shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -555,7 +555,8 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
     File dstFile = new File(dstPath.getPath());
     String randomSuffix =
         String.format(".%s_copyToLocal_", RandomStringUtils.randomAlphanumeric(8));
-    File tmpDst = new File(Name.HOME + randomSuffix);
+    File tmpDst = new File(PathUtils.concatPath( dstFile.getAbsolutePath(),
```

Comment:
@aaudiber  Do you think it is necessary to write a `getAbsolutePath` method in the `AlluxioURI`? Because the current `getPath` cannot parse relative path correctly. It would be a great pain if we lost the path information whenever parsing the path from user input.